securityscanutils/securityscan.go

```diff
@@ -41,6 +44,8 @@ type SecurityScanRepo struct {
 	// should be run through our scanner
 	scanReleasePredicate githubutils.RepositoryReleasePredicate
 
+	trivyScanner *TrivyScanner
```

Comment:
Now that we define a trivy scanner, could we mock it out and have tests that validate our whole job? This way we can validate the job without having to interact with trivy directly? Perhaps in a separate PR also